packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -193,18 +193,18 @@ class _DevToolsMenu extends StatelessWidget {
   }
 
   bool _shouldIncludeScreen(ScreenMetaData screen) {
-    // Some screens shouldn't show up in the menu.
-    return screen != ScreenMetaData.home &&
-        screen != ScreenMetaData.debugger &&
-        screen != ScreenMetaData.vmTools &&
-        // Or the generic "simple" screen.
-        screen != ScreenMetaData.simple &&
-        // DeepLink is currently behind a feature flag.
-        (screen != ScreenMetaData.deepLinks ||
-            FeatureFlags.deepLinkValidation) &&
-        // Or anything that requires some specific library that we can't
-        // check (`requiresLibrary` will be removed for extensions
-        // in future, and those will need some extra work to detect).
-        screen.requiresLibrary == null;
+    return switch (screen) {
+      // Some screens shouldn't show up in the menu.
+      ScreenMetaData.home => false,
+      ScreenMetaData.debugger => false,
+      ScreenMetaData.simple => false, // generic screen isn't a screen itself
+      // TODO(dantup): Check preferences.vmDeveloperModeEnabled
+      ScreenMetaData.vmTools => false,
```

Comment:
these cases can be joined with with the `||` operator (see my previous comment)

Reply:
I did that originally, but I didn't like how the formatter formatted it 😄

![image](https://github.com/flutter/devtools/assets/1078012/d5a874a7-e227-4c57-a2ac-93af7db42d1d)

If you think that's better though, I'll commit that before merging. Thanks!

Reply:
Doesn't matter too much. LGTM